labkey-client-api/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # The LabKey Remote API Library for Java - Change Log
 
+## version x.y.z
+*Released*: ??
+* Add Connection.setUserAgent() and Connection.getUserAgent()
+* Set Java library user agent to "LabKey Java API"
+* Set SAS library user agent to "LabKey SAS API"
```

Comment:
This belongs in the CHANGELOG.md for labkey-api-sas.

Reply:
> This belongs in the CHANGELOG.md for labkey-api-sas.

Well... technically, the SAS user agent change is part of the Java API. The `labkey-api-sas` folder contains just the SAS macros that call into our Java library. The code setting the user agent part of a shim layer in the Java API; it's there because SAS can only pass Strings and primitives as parameters.